scripts/ci/run-integration-tests

```diff
@@ -5,55 +5,183 @@
 # replace it with a test runner in the future, please see it more as a proof of
 # concept
 
-set -ex
+## Configuration
```

Comment:
I like the way you wrote the script and the effort you put into it. I really hope we can someday move this in the JS world to test it